private-templates-service/adaptivecards-templating-service/src/tests/router.test.ts

```diff
@@ -866,4 +866,66 @@ describe("Basic Get Templates", () => {
     // Check that only the latest version is returned
     expect(template.instances[0].version).toMatch("1.2");
   });
+
+  // Authenticated post request
+  it("pass in a data json and id and should receive a template with the data bound", async () => {
```

Comment:
Is it possible to add tests to bind data to a template with multiple versions? I think this test file should have a testUtils so you can call a helper function to create these templates for you.

Reply:
There's no testUtils page, but I added to the unit tests.